docs/architecture.md

```diff
@@ -6,7 +6,7 @@ The Pulp CLI architecture is described in this section.
 
 The Pulp CLI is designed with a plugin structure. Plugins can either live in the pulp-cli package or be shipped independently.
 By convention, all parts of the CLI are packages in the open namespace `pulpcore.cli`.
-A plugin can register itself with the main app by specifying by specifying its main module as a `pulp_cli.plugins` entrypoint in `setup.py`.
+A plugin can register itself with the main app by specifying its main module as a `pulp_cli.plugins` entrypoint in `setup.py`.
```

Comment:
Good catch, but we should start mentioning the `mount` command.

Reply:
You mean invert the order of the information, like first mention the `mount`  part and then the setup of the `entrypoint`?

Reply:
Oh, right. `mount` is already there. (It was invisible to the diff...)

Can you just replace "can" by "should" in line 19?